spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -23,6 +23,7 @@ spring:
       security:
         authorization:
           enabled: true
+          permit-all-paths: "/authenticate,/security/info,/features,/assets/**"
```

Comment:
Where is this used or who reads it?

Reply:
This is from the [config](https://github.com/spring-cloud/spring-cloud-common-security-config/blob/master/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/AuthorizationProperties.java#L46) and used in the auth configurations. For instance [this](https://github.com/spring-cloud/spring-cloud-common-security-config/blob/master/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/BasicAuthSecurityConfiguration.java#L93) is one of the instances.